src/app/ddDrakeModel.h

```diff
@@ -78,6 +84,10 @@ class DD_APP_EXPORT ddDrakeModel : public QObject
   static void addPackageSearchPath(const QString& searchPath);
   static QString findPackageDirectory(const QString& packageName);
 
+  #ifdef DRAKE_OH_FORK
+  void doKinematics(const QVector<double>& q, const QVector<double>& v, bool compute_gradients = false, bool compute_JdotV = true);
```

Comment:
yeah i would prefer to limit the scope of the api of ddDrakeModel now that the drake api is directly available in the python bindings